Azure/sonic-buildimage
# [Mellanox] Update FW binary to 29.2008.1958 for SPC-2 platforms

Signed-off-by: Volodymyr Samotiy <volodymyrs@nvidia.com>



**- Why I did it**
To update FW binary to 29.2008.1958 version for SPC-2 platforms

**- How I did it**
Updated FW binary version and appropriate submodule pointer

**- How to verify it**
Build an image, deploy and verify all is up and running

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

[Mellanox] Update FW binary to 29.2008.1958 for SPC-2 platforms
